it/it-tests/src/test/java/it/projectSearch/ProjectsPageTest.java

```diff
@@ -55,6 +57,11 @@ public static void setUp() {
     ORCHESTRATOR.executeBuild(SonarScanner.create(projectDir("duplications/file-duplications")).setProjectKey("key-bar"));
   }
 
+  @Before
+  public void before() {
+    clearBrowserLocalStorage();
```

Comment:
Yes we could, even my first tests failed. I'll do that in my branch feature/sb/it-refactoring.